app/controllers/boards_controller.rb

```diff
@@ -0,0 +1,6 @@
+class BoardsController < ApplicationController
+
+	def show
+	end
+
+end
```

Comment:
On a related note, spaces are generally preferred to tabs for indenting code because they're treated more consistently. For example, GitHub tends to render tabs as being 8 spaces wide, whereas Sublime seems to default to 4. This why your HTML and CSS files look much more deeply indented here.